jaxrs/src/main/java/org/jboss/as/jaxrs/JaxrsAttribute.java

```diff
@@ -0,0 +1,237 @@
+package org.jboss.as.jaxrs;
+
+import org.jboss.as.controller.SimpleAttributeDefinition;
+import org.jboss.as.controller.SimpleAttributeDefinitionBuilder;
+import org.jboss.as.controller.operations.validation.ModelTypeValidator;
+import org.jboss.dmr.ModelNode;
+import org.jboss.dmr.ModelType;
+
+/**
+ * Jaxrs configuration attributes.
+ *
+ * @author <a href="mailto:rsigal@redhat.com">Ron Sigal</a>
+ */
+public interface  JaxrsAttribute {
+
+    public static final String RESTEASY_PARAMETER_GROUP = "resteasy-parameter-group";
```

Comment:
Well, currently, RESTEasy 3.x gets configured by context-parameters, but I've proposed incorporating MicroProfile Config into 3.x, so that there would be multiple configuration sources. That's why I used the more generic "resteasy-parameter-group".

Reply:
I'm not sure to understand what you mean.
This attribute group is a hint to group related attributes together. If RestEasy would be configured with MicroProfile Config, what would be the impact on these attributes? Would there be another set of attributes with the "mp-config" attribute group?
As an aside, the attribute group name could simply be "resteasy"

Reply:
I would imagine that there would be one group consisting of exactly the RESTEasy parameters. I was just reacting to the **name** "context-parameters" since it would be too specific once RESTEasy starts using MP Config (although it turns out that that won't happen in WF 19). Just using "resteasy" sounds fine to me.